Runtime/Scripts/Settings/Data/MeshSyncPlayerConfig.cs

```diff
@@ -114,6 +114,7 @@ internal abstract class MeshSyncPlayerConfig : ISerializationCallbackReceiver {
     public bool UpdateMeshColliders       = true;
 
     [Obsolete]       public  bool                  SyncMaterials      = true;
+    
```

Comment:
Since there is no change in this file, can you please revert it ?

Reply:
Done